packages/error-reporting/README.md

```diff
@@ -76,9 +76,13 @@ process.on('uncaughtException', (e) => {
   // Report that same error the Stackdriver Error Service
   errors.report(e);
 });
+
+process.on('unhandledRejection', (reason, p) => {
+  errors.report('Unhandled rejection of promise: ' + p + ', reason: ' + reason);
```

Comment:
I don't think we should recommend concatenating the promise into this unless it has identifiable information.